apps/citizen_advocate/config/config.exs

```diff
@@ -22,6 +22,11 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:request_id]
 
+# additional keys `api_key` and `domain` are in .secret.exs
```

Comment:
mailgun because i happened to already have a mailgun account for some other random stuff, but this is pretty easy to switch out, lmk if you want creds to try this out